### PR TITLE
Fix hibernate CI

### PR DESCRIPTION
**Pull request by nvanbenschoten**
These three changes are needed to get CI working again with Hibernate, which is possible now that https://github.com/cockroachdb/cockroach-go/pull/24 is merged.

### CODE REVIEW

**Comment by cuongdo** [comment]
A lot of this Java stuff is foreign to me. 

@jordanlewis please help

File: testing/main_test.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func initORMApp(t *testing.T, app application, dbURL *url.URL) (killFunc, restar
 
 // waitForInit retries until a connection is successfully established.
 func waitForInit() error {
-	const maxWait = 15 * time.Second
+	const maxWait = 3 * time.Minute
```
Comment (cuongdo) [inline testing/main_test.go]: why so long?

Comment (nvanbenschoten) [inline testing/main_test.go]: Gradle+the JVM takes a while to start up sometimes (10s of seconds). This is super conservative, but I don't see much of a downside to that.

**Comment by nvanbenschoten** [comment]
Noted in the third commit that most of the changes were autogenerated by gradle.

**Pull request merged**